modules/filebeat/manifests/prospector.pp

```diff
@@ -30,12 +30,12 @@
   $exclude_lines         = [],
   $max_bytes             = '10485760',
   $multiline             = {},
-  $json                  = {},
+  $json                  = false,
   $tags                  = [],
   $symlinks              = false,
 ) {
 
-  validate_hash($fields, $multiline, $json)
+  validate_hash($fields, $multiline)
   validate_array($paths, $exclude_files, $include_lines, $exclude_lines, $tags)
   validate_bool($tail_files, $close_renamed, $close_removed, $close_eof, $clean_removed, $symlinks)
```

Comment:
For consistency with existing validating should $json be in this validate_bool grouping? (does seem somewhat inconsistent with most of our puppet to be validiating bools though)

Reply:
Oh yes, I totally missed that! May as well validate, since it's there...

Reply:
...and it turns out I forgot to change a couple of the (truthy) hashes, so it was useful in this case!